src/chrome/content/rules/Hauke-m.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="Hauke">
+	<target host="www.hauke-m.de" />
+	<target host="hauke-m.de" />
+
+	<securecookie host="^(www\.)?hauke-m\.de$" name=".*" />
+
+	<rule from="^http://(www\.)?hauke-m\.de/" to="https://$1hauke-m.de/" />
```

Comment:
Replace this line with

		<rule from="^http://(?:www\.)?hauke-m\.de/" to="https://www.hauke-m.de/" />

because `https://hauke-m.de/` redirects to `https://www.hauke-m.de/`.